README.md

```diff
@@ -0,0 +1,68 @@
+# hsk
+
+The `hsklogin` script uses the macOS ssh-agent with the yubikey, 
+so that you don't have to type in the PIN for every git pull.
+
+## Prerequirements
+
+Set up your yubikey for github SSH and test that it works correctly 
+(follow instructions at https://w3.ibm.com/#/support/article/hsk/github).
+
+
+## Installing
+
+```
+git clone https://github.com/mrszymon/hsk
+cd hsk
+sudo cp hsklogin /usr/local/bin
+chmod u+x /usr/local/bin/hsklogin
+```
+
+## Usage
```

Comment:
Suggestion: only put `hsklogin -h / --help` here.
Instead, enhance the actual help text within the tool itself. 

Every time you create documentation that doesn't sit together with the code, 
you create a breaking point ... that sooner or later will break, because somebody forgets to update the "external" documentation.